README.md

```diff
@@ -133,6 +123,15 @@ For models that have a many-to-many relationship, you can also pass the plural v
 Team.findAll({include: [Player]})
 ```
 
+Filtering by related models property, you can pass **where** paremeter inside each **include** item(s) object.
+```js
+// returns all team with their related City where City property name equals Healdsburg
+// GET /teams?include={"include": {"model": "City", "where": {"name": "Healdsburg"}}}
```

Comment:
Please get rid of the `include` key of the JSON passed to `include`, it's an unnecessary repetition.

It should look like this:
`/teams?include={"model": "City", "where": { "name": "Healdsburg" }}`